packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -26,14 +29,14 @@ const MT5TradeScreen = () => {
         () => ({
             ctrader: ctraderAccountsList,
             dxtrade: dxtradeAccountsList,
-            mt5: mt5AccountsList,
+            mt5: [mt5Account],
```

Comment:
Fixed a bug for trade modal where the title and landing company is not updated (still stuck to SVG even though we clicked on a BVI account), we need to pass in the account the user has clicked on, not check it from the `mt5accountslist`